include/IrcCore/ircdebug_p.h

```diff
@@ -142,8 +145,16 @@ static bool irc_debug_enabled(IrcConnection* c, uint l)
 
         dbg_init = true;
     }
-
-    return l <= dbg_level && (dbg_name.isEmpty() || QRegularExpression(QRegularExpression::wildcardToRegularExpression(dbg_name), QRegularExpression::CaseInsensitiveOption).match(c->displayName()).hasMatch());
+    if (l <= dbg_level && (dbg_name.isEmpty()))
+        return true;
+#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
```

Comment:
"if Qt version is _less than_ 5.15.0, then use QRegularExpression::wildcardToRegularExpression" doesn't seem right. the overload being used here was added in 5.12, so perhaps "=> 5.12.0"?